spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js

```diff
@@ -182,12 +207,11 @@ export class DialPad extends React.Component {
      */
     _onCountryCodeSelect(code) {
         this.setState({
-            selectedCountryCode: code,
+            defaultCountryCode: code,
+            parsedCountryCode: code,
```

Comment:
In this case we're getting the country code from the lib so we can trust its format?

Reply:
What do you mean? I'm not sure what to change here

Reply:
Do you want me to add a check if there's such country code? In this case it's coming from the country selector and not from the list, so it's predefined list.